src/App.vue

```diff
@@ -1,24 +1,66 @@
 <template>
-  <v-app>
-    <Navbar />
+  <v-app v-if="isLoggedIn">
+    <Navbar/>
       <v-container>
         <div class="content">
           <router-view />
         </div>
       </v-container>
   </v-app>
+  <v-app v-else>
+    <router-view v-if="isLoginTryCommited"/>
+  </v-app>
```

Comment:
Maybe you can only hide the navbar if the user is not logged in, router view will show the login page.
What is the purpose of isLoginTryCommited?

Reply:
The wrapping v-container and and div class ‘s css needs to be over written so the login page will be as planned

Reply:
You can conditionally use the class, if true use the class otherwise not

Reply:
Its not only the calss, its the v-container and the class so its a lot of conditions overall